src/js/menu/menu-item.js

```diff
@@ -51,15 +63,22 @@ class MenuItem extends Button {
    * @method selected
    */
   selected(selected) {
-    if (selected) {
-      this.addClass('vjs-selected');
-      this.el_.setAttribute('aria-selected',true);
-    } else {
-      this.removeClass('vjs-selected');
-      this.el_.setAttribute('aria-selected',false);
+    if (this.selectable) {
```

Comment:
Just a general thought, not necessary for this PR: would we have an easier time with menu-button/menu-item if we started from a `<select>` element?